tests/integration/test_api/test_config/test_max_upload_size/data/conf.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- tags:
+  - limitless_upload_size
+  configuration:
+    max_upload_size: 0
+- tags:
+  - low_upload_size
+  configuration:
+    max_upload_size: 5
```

Comment:
IMHO pytest tags are not the best option here. The configuration is the same, only values changed. In this case, we recommend adding some logic into the test in order to make it more scalable. Check out this example https://github.com/wazuh/wazuh-qa/blob/master/tests/integration/test_remoted/test_active_response/test_active_response_send_ar.py

Reply:
This test should include more cases. Maybe it could be a good idea to dynamically create the`test_cdb_list` with a specified size and refactor group creation to allow greater `max_upload_size`.